packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -1811,6 +1948,56 @@
         "type": "PathSet"
       }
     ],
+    [
+      "XChainClaimProof",
+      {
+        "nth": 1,
+        "isVLEncoded": false,
+        "isSerialized": true,
+        "isSigningField": true,
+        "type": "XChainClaimProof"
+      }
+    ],
+    [
+      "LockingChainIssue",
+      {
+        "nth": 1,
+        "isVLEncoded": false,
+        "isSerialized": true,
+        "isSigningField": true,
+        "type": "Issue"
+      }
+    ],
```

Comment:
Are these supposed to have the same `nth` value?

Reply:
As far as I understand it, the `nth` value must be unique for each type. These two fields' types are different, so it's fine that they have the same `nth` value.